### PR TITLE
Adds SHA-256 hasher.

### PR DESCRIPTION
**Pull request by coollog**
Related to #27

### CODE REVIEW

**Comment by loosebazooka** [comment]
I think we want to move away from `util` package naming. If it's a sha hasher, just throw it in `hash` or something like that.

**Comment by coollog** [comment]
Okay, changed the package to `hash`.

File: crepecake/src/main/java/com/google/cloud/tools/crepecake/hash/ByteHasher.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+package com.google.cloud.tools.crepecake.hash;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+/** Static utility class for generating SHA-256 hashes in hexadecimal. */
+public class ByteHasher {
+
+  private static final String SHA_256_ALGORITHM = "SHA-256";
+
```
Comment (loosebazooka) [inline crepecake/src/main/java/com/google/cloud/tools/crepecake/hash/ByteHasher.java]: Can you add a comment to this method?

Comment (loosebazooka) [reply crepecake/src/main/java/com/google/cloud/tools/crepecake/hash/ByteHasher.java]: oh it looks like it'll just be the same as the class header. Never mind.

**Pull request merged**